astropy/_erfa/erfa_generator.py

```diff
@@ -29,7 +29,7 @@
 NDIMS_REX = re.compile(re.escape("numpy.dtype([('fi0', '.*', <(.*)>)])").replace(r'\.\*', '.*').replace(r'\<', '(').replace(r'\>', ')'))
 
 
-class FunctionDoc(object):
```

Comment:
I am a bit sad to see this go, as I never did get used to new-style class using old-style syntax. 😢